thrift/server.cc

```diff
@@ -151,15 +154,21 @@ thrift_server::connection::process_one_request() {
     _output->resetBuffer();
     co_await read();
     ++_server._requests_served;
-    auto ret = _processor_promise.get_future();
+    auto ret = _processor_promise.get_future().handle_exception([&server = _server] (const std::exception_ptr&) {
+        server._requests_serving--;
+    });
```

Comment:
Nitpick: are the changes here to this function related to this patch? Maybe they should be folded into the previous patch?